plugins/techdocs-backend/README.md

```diff
@@ -6,16 +6,39 @@ This is the backend part of the techdocs plugin.
 
 This backend plugin can be started in a standalone mode from directly in this package
 with `yarn start`. However, it will have limited functionality and that process is
-most convenient when developing the catalog backend plugin itself.
+most convenient when developing the techdocs backend plugin itself.
 
-To evaluate the catalog and have a greater amount of functionality available, instead do
+To evaluate techdocs and have a greater amount of functionality available, instead do
```

Comment:
```suggestion
To evaluate TechDocs and have a greater amount of functionality available, instead do
```

Reply:
I think we can use "TechDocs" when we talk about the product and techdocs when its around a plugin or something that is named like that. Does it make sense?